src/elm/Pages/Build/View.elm

```diff
@@ -603,36 +604,50 @@ stepFollowButton stepNumber following =
 -}
 stepError : Step -> Html msg
 stepError step =
-    div [ class "step-error", Util.testAttribute "step-error" ]
-        [ span [ class "label" ] [ text "error:" ]
-        , span [ class "message" ]
-            [ text <|
-                if String.isEmpty step.error then
-                    "no error msg"
+    div [ class "message", class "error", Util.testAttribute "step-error" ]
+        [ span [] [ text "error:" ]
+        , text <|
+            if String.isEmpty step.error then
+                "null"
 
-                else
-                    step.error
-            ]
+            else
+                step.error
         ]
 
 
+{-| loadingLogs : renders message for loading logs
+-}
+loadingLogs : Html msg
+loadingLogs =
+    div [ class "message" ]
+        [ Util.smallLoaderWithText "loading..." ]
+
+
+{-| emptyLogs : renders message for empty logs
+-}
+emptyLogs : Html msg
+emptyLogs =
+    div [ class "message" ]
+        [ text "The build has not written logs to this step yet." ]
```

Comment:
Really minor but I noticed all the other messages are all in lowercase. Would it make sense for this one as well?

Reply:
great point, I agree @kneal 👍

Reply:
updated! 🚀